Umbraco-Cloud/Set-Up/Working-With-Visual-Studio/index.md

```diff
@@ -32,7 +32,15 @@ Once you have added a new project to the solution right-click on the solution in
 
 ![](images/solution-dependencies.png)  
 
-##Git - what should be committed
+## Using Umbraco namespaces in your *.Core project
```

Comment:
Instead of giving this a new heading, I think it should just be a note, descriping the current work-around for the bug (http://issues.umbraco.org/issue/UAAS-527)